Cannot find module '@babel/compat-data/corejs3-shipped-proposals'

After updating to `master` and running `yarn`, I cannot start `yarn serve`

```
 ERROR  Failed to compile with 1 errors                                                                                21:38:31
                                                                                                                               
 error  in ./src/main.js                                                                                                       
                                                                                                                               
Module build failed (from ./node_modules/babel-loader/lib/index.js):           
Error: [BABEL] /home/atd/dev/hestia-touch-one-ui/src/main.js: Cannot find module '@babel/compat-data/corejs3-shipped-proposals'
Require stack:                                                                                                                                     
- /home/atd/dev/hestia-touch-one-ui/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js                                           
- /home/atd/dev/hestia-touch-one-ui/node_modules/@babel/preset-env/lib/index.js                                                
- /home/atd/dev/hestia-touch-one-ui/node_modules/@vue/babel-preset-app/index.js                                                                    
- /home/atd/dev/hestia-touch-one-ui/node_modules/@babel/core/lib/config/files/plugins.js                                                           
- /home/atd/dev/hestia-touch-one-ui/node_modules/@babel/core/lib/config/files/index.js                                         
- /home/atd/dev/hestia-touch-one-ui/node_modules/@babel/core/lib/index.js                                                                          
- /home/atd/dev/hestia-touch-one-ui/node_modules/@vue/cli-plugin-babel/index.js
- /home/atd/dev/hestia-touch-one-ui/node_modules/@vue/cli-service/lib/Service.js                                               
- /home/atd/dev/hestia-touch-one-ui/node_modules/@vue/cli-service/bin/vue-cli-service.js (While processing: "/home/atd/dev/hestia-touch-one-ui/node_modules/@vue/babel-preset-
app/index.js")                                                                                                                                     
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:962:15)
    at Function.Module._load (internal/modules/cjs/loader.js:838:27)         
    at Module.require (internal/modules/cjs/loader.js:1022:19)
    at require (internal/modules/cjs/helpers.js:72:18)                                                                         
    at Object.<anonymous> (/home/atd/dev/hestia-touch-one-ui/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js:10:55)                                      
    at Module._compile (internal/modules/cjs/loader.js:1118:30)                                                                
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)                                                                      
    at Module.load (internal/modules/cjs/loader.js:982:32)                                                                                         
    at Function.Module._load (internal/modules/cjs/loader.js:875:14)                                                                               
    at Module.require (internal/modules/cjs/loader.js:1022:19)           
    at require (internal/modules/cjs/helpers.js:72:18)                                                                                             
    at Object.<anonymous> (/home/atd/dev/hestia-touch-one-ui/node_modules/@babel/preset-env/lib/index.js:29:44)                
    at Module._compile (internal/modules/cjs/loader.js:1118:30)                                                                                    
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1138:10)                                                                                                 
    at Module.load (internal/modules/cjs/loader.js:982:32)                             
    at Function.Module._load (internal/modules/cjs/loader.js:875:14)                                                                                                          
                                    
 @ multi (webpack)-dev-server/client?http://192.168.10.2:8080/sockjs-node (webpack)/hot/dev-server.js ./src/main.js    
```

--- a/package.json
+++ b/package.json
@@ -13,23 +13,23 @@
     "yarn": ">=1.12"
   },
   "dependencies": {
-    "core-js": "^3.6.4",
+    "core-js": "^3.6.5",
     "just-debounce": "^1.0.0",
     "mqtt": "^3.0.0",
-    "register-service-worker": "^1.6.2",
+    "register-service-worker": "^1.7.1",
     "typeface-roboto": "^0.0.75",
-    "vue": "^2.6.10",
-    "vuex": "^3.0.1"
+    "vue": "^2.6.11",
+    "vuex": "^3.4.0"
   },
   "devDependencies": {
-    "@vue/cli": "^4.2.3",
-    "@vue/cli-plugin-babel": "^4.2.3",
-    "@vue/cli-plugin-eslint": "^4.2.3",
-    "@vue/cli-plugin-pwa": "^4.2.3",
-    "@vue/cli-service": "^4.2.3",
-    "babel-eslint": "^10.0.1",
+    "@vue/cli": "^4.3.1",
+    "@vue/cli-plugin-babel": "^4.3.1",
+    "@vue/cli-plugin-eslint": "^4.3.1",
+    "@vue/cli-plugin-pwa": "^4.3.1",
+    "@vue/cli-service": "^4.3.1",
+    "babel-eslint": "^10.1.0",
     "eslint": "^5.16.0",
     "eslint-plugin-vue": "^5.0.0",
-    "vue-template-compiler": "^2.6.10"
+    "vue-template-compiler": "^2.6.11"
   }
 }

